<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFileDlg</class>
 <widget class="QDialog" name="AddFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="wLblTprz">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; color:#00aa00;&quot;&gt;图片日志&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wLetTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="wTetText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="wBtnSubmit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wBtnCancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="wDetDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="wLetPlace">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;地点&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wLblPlace">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>地点：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wBtnAddImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wBtnPreviewImage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图片预览</string>
   </property>
  </widget>
  <widget class="QLabel" name="wLblImageAmount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="wBtnDelete">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>401</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="wLbl1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>442</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>第</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wLetIndexDel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wLbl2">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>441</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>张</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddFileDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
